src/pages/Contactpage.jsx

```diff
@@ -1,5 +1,58 @@
+import { useState } from "react";
 import PageNav from "../components/PageNav";
+import { EMAIL_NOTIFICATION_ENDPOINT } from "../ApiEndpoints";
+
 function Contactpage() {
+  const [formData, setFormData] = useState({
+    email: "",
+    subject: "",
+    message: "",
+  });
+
+  const [statusMessage, setStatusMessage] = useState("");
+
+  // Handle form input change
+  const handleChange = (e) => {
+    setFormData({
+      ...formData,
+      [e.target.id]: e.target.value,
+    });
+  };
+
+  // Function to submit form data to the API
+  const handleSubmit = async (e) => {
+    e.preventDefault();
+
+    // Create payload to send to the backend
+    const payload = {
+      subject: formData.subject,
+      email: formData.email,
+      message: formData.message,
+    };
+
+    try {
+      const response = await fetch(
+        `${import.meta.env.VITE_BASE_URL}${EMAIL_NOTIFICATION_ENDPOINT}`,
```

Comment:
> `${import.meta.env.VITE_BASE_URL}` can you explain this ?? I am not sure how to store config in frontend
> 
> @aqueefcs Please check.

it is the base url of the api stored in env variable VITE_BASE_URL

Reply:
Yes I told you but Idk if this is the correct way ?

change name VITE_BASE_URL to BACKEND_BASE_URL

I am confirming with @aqueefcs how can I update this config while deploying.

Reply:
> Yes I told you but Idk if this is the correct way ?
> 
> change name VITE_BASE_URL to BACKEND_BASE_URL
> 
> I am confirming with @aqueefcs how can I update this config while deploying.

**we need to prefix VITE always while making any env variable in  vite+react  so it will be VITE_BACKEND_BASE_URL**

also the PR i made on the backend repo is  not reflecting in the gssoc leaderboard when asked they admin said the git hub repo PA changed the url of the repo without informing.

![image](https://github.com/user-attachments/assets/07d3cf7f-b457-4643-a2be-03c049b686a9)

Reply:
We haven't change anything.

Reply:
> We haven't change anything.

![image](https://github.com/user-attachments/assets/ee7ce3bc-0d84-426c-8355-e73c122eca57)
but my PR on globe hoppin backend is not showing and points are also not added